WebCryptoAPI/getRandomValues.js

```diff
@@ -42,6 +42,12 @@ function run_test() {
         }
     }, "Large length")
 
+    test(function () {
+        var arrayA = new Int8Array(10);
+        arrayB = self.crypto.getRandomValues(arrayA);
+        assert_true( arrayA == arrayB, "arrays match");
```

Comment:
Please use `assert_equals`, and avoid adding a space after the opening parenthesis.